.github/workflows/signable-rspec.yml

```diff
@@ -0,0 +1,34 @@
+name: CI RSpec Tests
+
+on:
+  push:
+    paths:
+    - 'signable/**'
+  pull_request:
+    paths:
+    - 'signable/**'
+
+jobs:
+  build:
+    name: CI
+    runs-on: ubuntu-latest
+    env:
+      api-dir: ./signable
+    steps:
+      - uses: actions/build@v3
+      - uses: actions/checkout@v1
+      - uses: actions/setup-ruby@v1
+    with:
+      ruby-version: 2.3.0
+
+  - name: Run bundle install
+    working-directory: ${{env.api-dir}}
+    run: |
+      gem install bundler
+      bundle install --jobs 4 --retry 3
+
+  - name: Build and test with rspec
+    working-directory: ${{env.api-dir}}
+    env:
+      RAILS_ENV: test
+    run: bundle exec rspec
```

Comment:
Not sure but I think we might need to indent these to the same level as `- uses: `